orchagent/portsorch.cpp

```diff
@@ -3319,6 +3335,48 @@ bool PortsOrch::addBridgePort(Port &port)
     sai_attribute_t attr;
     vector<sai_attribute_t> attrs;
 
+    if (port.m_type == Port::PHY)
```

Comment:
can it be convereted to switch stmt instead if-else?

Reply:
Original code also had if else and hence retained it.